services/app-web/src/components/Header/UserLoginInfo/UserLoginInfo.tsx

```diff
@@ -23,22 +25,83 @@ const LoggedInUserInfo = (
     logout: LogoutHandlerT
 ): React.ReactElement => {
     const stringConstants = useStringConstants()
-    return (
-        <div className={styles.userInfo}>
-            <span>Contact </span>
-            <ContactSupportLink alternateText={stringConstants.MAIL_TO_SUPPORT} />
-            <span className={styles.divider}>|</span>
-            <span>{user.email}</span>
-            <span className={styles.divider}>|</span>
+    const [isOpen, setIsOpen] = useState(false)
+    const { logButtonEvent } = useTealium()
+    const onToggle = (): void => {
+        logButtonEvent({
+            text: 'Your account',
+            button_type: 'button',
+            button_style: 'unstyled',
+            parent_component_type: 'constant header',
+        })
+        setIsOpen((prevIsOpen) => {
+            return !prevIsOpen
+        })
+    }
 
-            <ButtonWithLogging
-                type="button"
-                unstyled
-                parent_component_type="constant header"
-                onClick={logout}
-            >
-                Sign out
-            </ButtonWithLogging>
+    const testMenuItems = [
```

Comment:
don't define JSX inside the body of a function, it messes up clean/dirty tracking and will always be remounted.